index.html

```diff
@@ -1,51 +1,111 @@
-          <label>Subtext:</label>
-          <input id="subtext" placeholder="Your subtext here" type="text" name="subtext" autocomplete="off" required>
-          <input type="submit" id="generateBTN" value="Generate">
-      </form>
-      <h2>↓ Your Generated Banner will appear below after a few seconds ↓</h2>
-      <hr>
-      <div class="bannerContainer">
-        <img id="banner" src="res/images/examplebanner.png" alt="Your Banner">
-        <div class="bannerOverlay errorOverlay">
-          <p>
-            Cannot generate banner
-            <br />
-            Username correct?
-          </p>
-        </div>
-        <div class="bannerOverlay spinnerOverlay">
-          <div class="spinner">Loading..</div>
-        </div>
-      </div>
-      <hr>
-      <a title="Check out the cool contributers!" href="https://devcredits.herokuapp.com/?title=devBanner&users=cozyplanes,Kimmax,mitiko,ewpratten"><p class="info">Credits</p></a>
-      <br />
-      <p class="info">Want to subscribe to <b>Telegram</b> for updates? Visit <a target="_blank" href="https://t.me/devbanner">Here</a></p>
-      <p class="info">Want to contribute to the <b>frontend</b> on Github? Visit <a target="_blank" href="https://github.com/devBanner/devBanner_Frontend">Here</a></p>
-      <p class="info">Want to contribute to the <b>backend</b> on Github? Visit <a target="_blank" href="https://github.com/devBanner/devBanner">Here</a></p>
-      <p class="info">Want to contribute or share ideas on <b>Gitter</b> chat? Visit <a target="_blank" href="https://gitter.im/devbanner/Lobby?utm_source=share-link&utm_medium=link&utm_campaign=share-link">Here</a></p>
-      <p class="info">Want to see our <b>FAQ</b>? Visit <a target="_blank" href="https://devbanner.gitbooks.io/faq">Here</a></p>
-    </div>
-    <script type="text/javascript" src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
-    <script type="text/javascript" src="res/js/script.js"></script>
-  </body>
-</html>
+<html lang="en">
+<head>
+	<meta charset="utf-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>devBanner - Create Your Own devRant Banner!</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<link rel="shortcut icon" type="image/x-icon" href="res/images/favicon.ico">
+
+	<link rel="stylesheet" type="text/css" media="screen" href="lib/Bootstrap/bootstrap.min.css">
+	<link href="https://fonts.googleapis.com/css?family=Comfortaa:400,700" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" media="screen" href="lib/AnimateCSS/animate.min.css" />
+	<link rel="stylesheet" type="text/css" media="screen" href="res/css/stylesheet.css" />
+	<link rel="stylesheet" type="text/css" media="screen" href="res/css/spinner.css" />
+</head>
+<body>
+<div class="content">
+	<div id="header">
+		<span id="header-before" style="background-color: #9e85a1;"></span>
+		<img id="header-img" class="img-fluid" src="res/images/logo_transparent.png">
```

Comment:
Licensing of the header image, as mentioned above

Reply:
Well devRantron uses it, DevRant-Stats uses it, you use it now.
Shouldn't be a problem!